pkg/cmd/version/version.go

```diff
@@ -30,17 +30,19 @@ import (
 	"github.com/tektoncd/cli/pkg/version"
 )
 
-// NOTE: use go build -ldflags "-X github.com/tektoncd/cli/pkg/cmd/version.clientVersion=$(git describe)"
-var clientVersion = devVersion
-
-// flag to skip check flag in version cmd
-var skipCheckFlag = "false"
-
 const (
 	devVersion       = "dev"
 	latestReleaseURL = "https://api.github.com/repos/tektoncd/cli/releases/latest"
 )
 
+var (
+	// NOTE: use go build -ldflags "-X github.com/tektoncd/cli/pkg/cmd/version.clientVersion=$(git describe)"
+	clientVersion = devVersion
+	// flag to skip check flag in version cmd
+	skipCheckFlag = "false"
+	namespace     string
```

Comment:
@vdemeester I mean we can put the default value here, and this can be overridden with LDFLAG and also namespace flag in cmd. And we can further use this in all functions. This way we can remove the defaultNamespace flag from pkg cc @savitaashture

Reply:
This can be single source for namespace then rather than two different for cmd and LDFLAG

Reply:
@piyush-garg updated code to have same var for both cmd and LDFLAG but not for default because now default is an []string{} and as per my understanding LDFLAG doesn't take the `[]` .
Still trying if something can be done to make it single var